nbs/examples/Getting_Started.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>I think the correct syntax is NeuralForecast.models. And you don't need to specify the core module in the NeuralForecast.core.</p><p> </p><p>E.g: </p><p>This notebook provides an example on how to start using the main functionalities of the NeuralForecast library. The <code>NeuralForecast</code> class allows users to easily interact with <code>NeuralForecast.models</code> PyTorch models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>